src/inspector/node_protocol.pdl

```diff
@@ -92,3 +92,12 @@ experimental domain NodeWorker
       # Identifier of a session which sends a message.
       SessionID sessionId
       string message
+
+experimental domain NodeRuntime
+  command enable
+  command disable
+
+  # Event is issued when Node process is finished and waiting for debugger to disconnect.
+  # When domain is enabled client should listen for this event instead of `Runtime.executionContextDestroyed`
+  # to check wether node process is finished.
+  event waitingForDebuggerToDisconnect
```

Comment:
This is slightly misleading (this notification is sent when Node is waiting for all the inspector clients to disconnect but inspector clients are not necessarily "debuggers"). I understand the need to be consistent with the text printed to the console. Note that the messages are intended for the user while this notification is sent to the frontend. Maybe just `waitingForDisconnect`?

Reply:
done!